jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -113,9 +114,13 @@ private static boolean loadNativeLibrary() {
 			try {
 				AccessController.doPrivileged((PrivilegedAction<Void>) () -> {
 					System.loadLibrary("j9criu29"); //$NON-NLS-1$
-					nativeLoaded = true;
 					return null;
 				});
+				if (setupJNIFieldIDsAndCRIUAPI()) {
+					nativeLoaded = true;
+				} else {
+					errorMsg = "setupJNIFieldIDsAndCRIUAPI() failed"; //$NON-NLS-1$
```

Comment:
you dont need to set this, its automatically set in getErrorMEssage

Reply:
Right, removed.